tabpfn_client/server_config.yaml

```diff
@@ -8,6 +8,7 @@ protocol: "https"
 host: "tabpfn-server-wjedmz7r5a-ez.a.run.app"
 # host: tabpfn-server-preprod-wjedmz7r5a-ez.a.run.app   # preprod
 port: "443"
```

Comment:
Before merging the PR change the logo and title of the UI

Reply:
Done: https://frontend-1039906307296.europe-west4.run.app/